### PR TITLE
[zh] Update content\zh-cn\docs\concepts\_index.md

### PR DESCRIPTION
**Pull request by ydFu**
Update this page to upstream, it is miss `<glossary_tooltip >`  label now.

```
content\zh-cn\docs\concepts\_index.md
```
Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
@kubernetes/sig-docs-zh-owners

File: content/zh-cn/docs/concepts/_index.md [hunk 1]
```diff
@@ -17,5 +17,8 @@ weight: 40
 <!--
 The Concepts section helps you learn about the parts of the Kubernetes system and the abstractions Kubernetes uses to represent your {{< glossary_tooltip text="cluster" term_id="cluster" length="all" >}}, and helps you obtain a deeper understanding of how Kubernetes works.
 -->
+概念部分帮助你了解 Kubernetes 系统的各个部分以及 Kubernetes 用来表示
+{{<glossary_tooltip text="集群" term_id="cluster" length="all" >}}
+的抽象概念，并帮助你获得更深入的理解 Kubernetes 是如何工作的。
```
Comment (howieyuen) [inline content/zh-cn/docs/concepts/_index.md:22]: 因为 glossary_tooltip 的 text 字段是中文，所以这里单独成行，渲染的结果会在“集群”前后各有一个空格；
```suggestion
概念部分帮助你了解 Kubernetes 系统的各个部分以及
Kubernetes 用来表示{{<glossary_tooltip text="集群" term_id="cluster" length="all" >}}的抽象概念，
并帮助你获得更深入的理解 Kubernetes 是如何工作的。
```

Comment (tengqm) [inline content/zh-cn/docs/concepts/_index.md:22]:
```suggestion
的抽象概念，并帮助你更深入地理解 Kubernetes 是如何工作的。
```

**Comment by howieyuen** [comment]
PR has conflict...

**Pull request merged**